source/agora/consensus/protocol/Nominator.d

```diff
@@ -409,14 +412,17 @@ extern(D):
             return;
         }
 
-        if (this.nomination_start_time == 0)
+        if (!this.is_nominating)
```

Comment:
> I think this is an unrelated change.

it is related, I had to start using that boolean to decide if the nomination round has started or not to be consistent with the code a couple of lines below

Reply:
this line was added because of the subset change:
https://github.com/bosagora/agora/pull/2235/files#diff-2c5fcbf7251215d5e7aba3eaa09c23da9d88c3645e5c278b9ef351b9798184f7R423

and the line you commented was changed to be consistent with the line I referenced above